spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -42,5 +42,34 @@
 			<artifactId>spring-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- SI channel adapters -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.api</groupId>
+			<artifactId>api-common</artifactId>
+			<!-- This shouldn't be required here. -->
+      <version>1.0.0</version>
+    </dependency>
 	</dependencies>
+
+	<!-- Must also get rid of this -->
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>animal-sniffer-maven-plugin</artifactId>
```

Comment:
This plugin is being invoked by some dependency that I couldn't locate, to inspect language level. In this case, it was failing the build because it found Java 8 constructs and it was configured for language level 7.